.travis/deploy-storybook.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eu
+
+GS_BUCKET_NAME=sentryio-storybook
+DEPLOY_BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
+echo "Build branch: ${DEPLOY_BRANCH}"
+
+# Transform the branch name to a bucket directory name, give preference to "master"
+if [ "${DEPLOY_BRANCH}" = "master" ]; then
+  BUCKET_DIR_NAME="_master"
+else
+  BUCKET_DIR_NAME=$(echo "${DEPLOY_BRANCH}" | tr '[:upper:]./' '[:lower:]--' | tr -cd '[:alnum:]-_')
```

Comment:
Can you tell me what this does with the branch name? The common non-alphanumeric characters we use are `/`, `-`, `_`.

Reply:
Oops just re-read your PR